maro/simulator/scenarios/citi_bike/topologies/ny.201804/config.yml

```diff
@@ -25,11 +25,6 @@ decision:
         windows: 10 # frame index number for the window
         num: 20
 
-reward: # reward options
```

Comment:
double confirm with @chaosddp

Reply:
It is ok to delete these from config files as we deleted reward calculation